### ISSUE TITLE
background music based on game and other suggestions

### ISSUE THREAD

**Issue by foss-lover34** [saVantCZ/PS5reborn#29]
Hello, awesome theme as always and nice updates. i wanted to ask how to add background music based on the game like official ps5 like the one for gta 5 is this 
[gta5background.mp3.zip](https://github.com/user-attachments/files/17895564/gta5background.mp3.zip)

as well as selecting which items appear in the welcome hub
 with the option to disable them, and to treat it like a game so i can give it a video as background all so i can have the background loop with the particle really blend in good, on the welcome hub **and making it the default item when launching the theme**

I also wanted to ask to remove windows media player dependency, i dont have it and i cant install it and i tried changing windows codec but the startup video doesnt work, i dont need that anymore even if it would be awesome because of this simple windows script that looks like this:

"start "" "E:\Apps\ffplay\ffplay.exe" -left 0 -top 0 -alwaysontop -noborder -autoexit -loglevel quiet -loop 1 "E:\Apps\Playnite\intro.mkv" 2>NUL

timeout 9

start "" "E:\Apps\Playnite\Playnite.fullscreenapp.exe" --hidesplashscreen"

and i was very happy until i discovered that if i disable the native plugin intro video, and disable the "press x to continue" so that after the video manually played by cmd, playnite opens directly into the library of games, sadly it has this problem that the "press x to continue" text still appears even if for a split second and quicly dismisses itself, and that breaks my animations fade between the video and the starting background that appears right after the black screen of "press x" like in this video

https://github.com/user-attachments/assets/c85cb696-c822-4c16-a76f-224d9f194913
on this video you can see also that the moving particles suddendly disappear, even if the "press x" didnt appear, even if the light fades in perfectly, did you remove the particles from the background?
 thats why i suggested the option to add a background video to the welcome hub so that i can create a video to play immediatly when playnite launches with this theme and have it transition with the startup video. Or you could edit the default background to be with the particles before swapping to the video of the game (in this video example i didnt configure a video or even a background for sparkingzero so thats why it uses that one)

You could also add a always playing background music like the one the ps5 has and it would be dope if it combined them like in this video! https://www.youtube.com/watch?v=jc6zzjOoo3Y

I hope i made myself clear in this post because im very passionate about this and i hope that at least something in here will give you some ideas 👍 :)

**Comment by saVantCZ** [comment]
You can add game music with plugin "Playnite Sounds Mod" 

Its not possible to rearrange welcome hub items, but you can enable video background on welcome hub by enabling it in Theme Options plugin, also you can replace its video file in theme folder (called explorebackground.mp4)

Also its not possible to make it default selected item on main screen because of how playnite works, it will always keep game list on selected game and keep it focused.

I cannot remove windows media player dependency, its not up to me because WPF mediaelement element as part of net framework is exclusively using windows media player. If you want to use playnite with video, like intro, background video and trailers you need to use windows media player, there is no other way of playing videos.

All fade animations for nicely fading from intro to particles and other way is dependent on windows media player, again if you want to fix these transitions, install media player.

Windows media player is part of windows by default... no idea why you dont wanna keep it installed, it doesnt interfere with anything and i reccomend to keep it installed.

There is always playing PS5 background ambient music, if game has no game music set by Playnite Sounds Mod plugin it will play as default, plus any other submenu like playnite settings will play ps5 ambient music

Long story short, windows media player is default app in windows, keep it. You will loose so much without it, theme/playnite is WPF application and needs windows files. And its still just a theme for browsing games, not PS5 simulator.

**Comment by foss-lover34** [comment]
so theres no way to remove the fake windgets fro the welcome hub even tough you can edit all the text of it? i dont want to rearrange them just to disable some / all of them.

For playnite esound music mod i didnt find any way to add a different background music when a game is highlighted and do it for every game how to do that?

**Comment by saVantCZ** [comment]
Once you install playnite sounds mod plugin, set it up on desktop mode, extensions. Then you can select all games in desktop mode, right click, select playnite sounds and dowload audio via YouTube. You can manualy select music for games or let it select automatically

**Comment by saVantCZ** [comment]
And i will take a look on disabling some widgets on welcome page

**Comment by saVantCZ** [comment]
Latest update add ability to disable each widget in welcome hub.

Hope you resolve rest which are not theme related.
